cmd/svcat/class/describe_cmd.go

```diff
@@ -47,44 +56,56 @@ func NewDescribeCmd(cxt *command.Context) *cobra.Command {
 		RunE:    command.RunE(describeCmd),
 	}
 	cmd.Flags().BoolVarP(
-		&describeCmd.lookupByKubeName,
+		&describeCmd.LookupByKubeName,
 		"kube-name",
 		"k",
 		false,
-		"Whether or not to get the class by its Kubernetes Name (the default is by external name)",
+		"Whether or not to get the class by its Kubernetes name (the default is by external name)",
 	)
+	describeCmd.AddScopedFlags(cmd.Flags(), true)
+
```

Comment:
Yeah, the reason the test passes is the actual bit that compiles the command and parses junk is inside the cobra framework, so I can't test it. I have to manually reach in and set the fields.

As for filtering the plans by namespace, that shouldn't be necessary because we're filtering by the exact k8s name of the parent class (filtered through the pkg/svcat interface so it doesn't matter if it's a ServiceClass or ClusterServiceClass).

Reply:
> As for filtering the plans by namespace, that shouldn't be necessary because we're filtering by the exact k8s name of the parent class (filtered through the pkg/svcat interface so it doesn't matter if it's a ServiceClass or ClusterServiceClass).

I was doing some poking a week ago with this review as a non-admin user and specified a single namespace on the describe class command.  I got errors indicating I didn't have privs to get **cluster scoped plans**.  So I think this is a real issue, but I'll have to wait 'til next time I have an environment up and time to reproduce.  I'm fine if we address that in a separate issue.

Reply:
oh huh, that is interesting. Although I think that's a larger problem that the scope of this PR, because doesn't that mean you wouldn't be able to see cluster plans to know what they're called to even provision one?